openem-ingestor/service/openem-ingestor-config.yaml

```diff
@@ -0,0 +1,13 @@
+Scicat:
+  Host: http://openem-scopem.ethz.ch:89/api/v3
+  AccessToken: "asdf"
+Transfer:
+  Method: S3
+  S3:
+    Endpoint: scopem-openem.ethz.ch:9000
```

Comment:
If this and the host above are real endpoints, you should probably change them to something imaginary. This is only an example config after all!

Reply:
Sure. I used real config values in order to use it during development. The debug task copies this file next to the executable. I added my debug config file to userConfigDir()/openem-ingestor/ which takes precedent, so this can now just be an example.